web/modules/custom/joinup_core/joinup_core.install

```diff
@@ -60,3 +60,41 @@ function joinup_core_requirements($phase) {
 function joinup_core_update_8100(&$sandbox) {
   \Drupal::service('module_installer')->install(['config_sync']);
 }
+
+/**
+ * Use the 'private' scheme for contact form uploads.
+ */
+function joinup_core_update_8101() {
+  include_once DRUPAL_ROOT . '/core/includes/install.inc';
+  $settings_file = DRUPAL_ROOT . '/sites/default/settings.php';
+
+  // The private files directory should have been already created.
+  if (!$private_files_directory = realpath(DRUPAL_ROOT . '/../private')) {
+    throw new \RuntimeException("Directory '$private_files_directory' has to be created prior running this script.");
+  }
+
+  // As the private files directory is outside the web tree, we really don't
+  // need a .htaccess file but we still provide it to comply with the Drupal
+  // security policy in the unlikely case this directory will be moved under the
+  // web tree in the future.
+  file_save_htaccess($private_files_directory);
```

Comment:
I wouldn't mess with this stuff in an update hook. This is not the responsibility of the application, but of the system administrators.